guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -30,10 +30,13 @@ Use this procedure to enable the repositories that are required to install {Prod
 ----
 # dnf module enable {dnf-module}
 ----
+
 +
 [NOTE]
 ====
-include::snip_dnf-module-enable-note.adoc[]
+If there is any warning about conflicts with Ruby or PostgreSQL while enabling `{dnf-module}` module, see {InstallingServerDocURL}Troubleshooting_Ruby_{project-context}[Troubleshooting Ruby] or
+{InstallingServerDocURL}Troubleshooting_Postgresql_{project-context}[Troubleshooting PostgreSQL].
```

Comment:
@asteflova Can you please check if this works in your downstream? I vaguely remember a similar PR from you with a different approach.

Reply:
Works for me (by replacing the version in the URL with the current one) :+1: